tests/functional/curate/cram/parse-genbank-location/empty-location.t

```diff
@@ -0,0 +1,10 @@
+Setup
+
+  $ export AUGUR="${AUGUR:-$TESTDIR/../../../../../bin/augur}"
+  $ echo -e '{"geo_loc_name":"Canada:Vancouver"}\n{"geo_loc_name":""}' > test.ndjson
+
+Records without a `location` field result in the expected warning
```

Comment:
Test docs here don't match the actual test below

```suggestion
Records with an empty `location` field pass through without warning
```

Reply:
force-pushed a correction.